lib/install/planbuilder.go

```diff
@@ -349,14 +351,29 @@ func (b *PlanBuilder) AddRBACPhase(plan *storage.OperationPlan) {
 	})
 }
 
-// AddResourcesPhase appends K8s resources initialization phase to the provided plan
-func (b *PlanBuilder) AddResourcesPhase(plan *storage.OperationPlan) {
+// AddSystemResourcesPhase appends phase that creates system Kubernetes
+// resources to the provided plan.
+func (b *PlanBuilder) AddSystemResourcesPhase(plan *storage.OperationPlan) {
+	plan.Phases = append(plan.Phases, storage.OperationPhase{
+		ID:          phases.SystemResourcesPhase,
+		Description: "Create system Kubernetes resources",
+		Data: &storage.OperationPhaseData{
+			Server:  &b.Master,
+			Cluster: &b.Cluster,
```

Comment:
Why not store cluster record directly in the operation plan? I'm sure it's useful in other places and will even minimize redundancy.

Reply:
I'm actually not sure why I did that at all. ClusterID is available in the plan so I can just query it.